### PR TITLE
Add review for Portuguese translation of `softwarereview_reviewer.pt.Rmd`

### PR DESCRIPTION
**Pull request by pedropark99**
This PR is adding the first review of `softwarereview_reviewer.pt.Rmd`.

### CODE REVIEW

**Comment by arianacabral** [comment]
I can conduct the second review of this chapter.

**Comment by pedropark99** [comment]
Hey @arianacabral ! Thank you very much for your interest! If you want to, you can perform this second review by simply reviewing the changes in this PR 😉.

You review the changes by going into the "Files changed" panel in this PR. In other words, go to this link and review the changes: https://github.com/ropensci/dev_guide/pull/851/files

**Comment by arianacabral** [comment]
> Hey @arianacabral ! Thank you very much for your interest! If you want to, you can perform this second review by simply reviewing the changes in this PR 😉.
> 
> You review the changes by going into the "Files changed" panel in this PR. In other words, go to this link and review the changes: https://github.com/ropensci/dev_guide/pull/851/files

Hi, @pedropark99! 

Thank you for providing the details! 

I’ll review the changes in the "Files changed" panel and will share my feedback as soon as possible 😊

File: softwarereview_reviewer.pt.Rmd
Comment (arianacabral) [inline softwarereview_reviewer.pt.Rmd:73]: Acredito que talvez faça mais sentido, para quem está lendo, manter essa estrutura. Se achar adequado, pode alterar as demais para que fiquem todos os exemplos no mesmo padrão.

```suggestion
- [revisão 1](https://github.com/ropensci/software-review/issues/191#issuecomment-367166658) e [revisão2](https://github.com/ropensci/software-review/issues/191#issuecomment-368254623) de `rtika`
```

**Review by arianacabral — CHANGES_REQUESTED** [review]
Hi, @pedropark99!

I have completed the second review.

I must say that I was very impressed with the first review; it was very good.

In this second review, I made just a few minor adjustments to improve some sentences and to standardize the reference to certain terms. However, please feel free to modify any of my suggestions if you think it’s necessary. 

For some points where I had doubts, I’ve left comments.

File: softwarereview_reviewer.pt.Rmd
Comment (pedropark99) [reply softwarereview_reviewer.pt.Rmd:12]: I just noticed the lack of "(as)" in this other instance.

```suggestion
Tente ao máximo concluir sua avaliação em até 3 semanas após o aceite da solicitação de avaliação. Nós vamos lembrar os(as) revisores(as) das datas de vencimento, tanto futuras quanto passadas. Os(as) editores(as) podem designar revisores(as) adicionais ou alternativos se uma revisão estiver muito atrasada.
```

File: softwarereview_reviewer.pt.Rmd
Comment (pedropark99) [reply softwarereview_reviewer.pt.Rmd:28]: O termo "issue" está sendo mantido intacto. Pois a tradução direta "problema" é muito esquisita, e não ajuda muito a entendermos o significado original (isto é, uma issue do GitHub).

File: softwarereview_reviewer.pt.Rmd
Comment (pedropark99) [reply softwarereview_reviewer.pt.Rmd:37]: Boa! Excelente essa mudança

File: softwarereview_reviewer.pt.Rmd
Comment (pedropark99) [reply softwarereview_reviewer.pt.Rmd]: Faz sentido mesmo mudar essa estrutura dessa forma.

**Comment by pedropark99** [comment]
First of all, thank you very much @arianacabral for your review!

@yabellini  the two reviews are done, we can proceed with this PR.

**Pull request merged**
